src/TributeEvents.js

```diff
@@ -114,7 +114,7 @@ class TributeEvents {
             return
         }
 
-        if (!instance.tribute.isActive) {
+        if (true) {
```

Comment:
@rcavezza We can probably just remove this conditional since `true` is always `true`.